app/javascript/packages/document-capture/context/native-camera-a-b-test.tsx

```diff
@@ -0,0 +1,47 @@
+import { createContext } from 'react';
+import type { ReactNode } from 'react';
+
+interface NativeCameraABTestContextValue {
+  /**
+   * Whether or not the A/B testing of the native camera only option is enabled.
+   */
+  nativeCameraABTestingEnabled: boolean;
+  /**
+   * Whether or not the Acuant SDK should be bypassed and only the native camera
+   * offered to the user.
+   */
+  nativeCameraOnly: boolean;
+}
+
+const NativeCameraABTestContext = createContext<NativeCameraABTestContextValue>({
+  nativeCameraABTestingEnabled: false,
+  nativeCameraOnly: false,
+});
+
+NativeCameraABTestContext.displayName = 'NativeCameraABTestContext';
+
+interface NativeCameraABTestContextProviderProps {
+  children: ReactNode;
+  nativeCameraABTestingEnabled: boolean;
+  nativeCameraOnly: boolean;
+}
+
+function NativeCameraABTestContextProvider({
+  children,
+  nativeCameraABTestingEnabled,
+  nativeCameraOnly,
+}: NativeCameraABTestContextProviderProps) {
+  return (
+    <NativeCameraABTestContext.Provider
+      value={{
+        nativeCameraOnly,
+        nativeCameraABTestingEnabled,
+      }}
+    >
+      {children}
+    </NativeCameraABTestContext.Provider>
+  );
+}
```

Comment:
Eric and I paired a bit on this last Friday and I couldn't (easily) get that change working? I'm not terribly concerned since this will be a short lived context - we will revert this code once the test is complete - so unless you have objections, I'm going to leave it as is.

Reply:
Yeah, mostly a simplification refactoring, but if this is temporary, not worth concerning too much over.